osx/mpkg/quickstart.app/Contents/Resources/Scripts/start.sh

```diff
@@ -29,7 +29,17 @@ VM_EXISTS_CODE=$?
 if [ $VM_EXISTS_CODE -eq 1 ]; then
   "${DOCKER_MACHINE}" rm -f "${VM}" &> /dev/null
   rm -rf ~/.docker/machine/machines/"${VM}"
-  "${DOCKER_MACHINE}" create -d virtualbox --virtualbox-memory 2048 --virtualbox-disk-size 204800 "${VM}"
+  #set proxy variables if they exists
+  if [ ! -z ${HTTP_PROXY+x} ]; then
+	PROXY_ENV="$PROXY_ENV --engine-env HTTP_PROXY='$HTTP_PROXY'"
+  fi
+  if [ ! -z ${HTTPS_PROXY+x} ]; then
+	PROXY_ENV="$PROXY_ENV --engine-env HTTPS_PROXY='$HTTPS_PROXY'"
+  fi
+  if [ ! -z ${NO_PROXY+x} ]; then
+	PROXY_ENV="$PROXY_ENV --engine-env NO_PROXY='$NO_PROXY'"
+  fi  
+  "${DOCKER_MACHINE}" create -d virtualbox $PROXY_ENV --virtualbox-memory 2048 --virtualbox-disk-size 204800 "${VM}"
```

Comment:
Unless this uses `eval`, the quoting inside `PROXY_ENV` likely won't work the way you expect it to (I'd think it's probably sending literal `'xxx'` to `docker-machine` including the quotes); that being said, I can't imagine folks with a `PROXY` variable are actually using spaces, so we're probably safe enough to just drop those.